src/components/StudyView/StudyView.tsx

```diff
@@ -128,6 +139,11 @@ const StudyView: React.FC<StudyViewProps> = ({ submissionId }) => {
           <SampleList
             submission={submission.data}
             onSampleCreate={handleSampleCreate}
+            sampleCreateFailureMessage={
+              lockMutation.isError
+                ? `Cannot create new sample because this study is currently being edited by ${submission.data.locked_by?.name || "unknown user"}`
```

Comment:
Suggestion (not requirement). I'll defer to others for the final decision here (i.e. keep it how it is or add "an").

```suggestion
                ? `Cannot create new sample because this study is currently being edited by ${submission.data.locked_by?.name || "an unknown user"}`
```